src/Vm.sol

```diff
@@ -188,156 +249,135 @@ interface VmSafe {
-    // (stringified_json, path)
-    function writeJson(string calldata, string calldata) external;
+    // json1 and json2 are simply keys used by the backend to keep track of the objects. So vm.serializeJson(json1,..)
+    // will find the object in-memory that is keyed by "some key".
+    function writeJson(string calldata json, string calldata path) external;
     // Write a serialized JSON object to an **existing** JSON file, replacing a value with key = <value_key>
     // This is useful to replace a specific value of a JSON file, without having to parse the entire thing
-    // (stringified_json, path, value_key)
-    function writeJson(string calldata, string calldata, string calldata) external;
+    function writeJson(string calldata json, string calldata path, string calldata valueKey) external;
     // Returns the RPC url for the given alias
-    function rpcUrl(string calldata) external view returns (string memory);
+    function rpcUrl(string calldata rpcAlias) external view returns (string memory json);
     // Returns all rpc urls and their aliases `[alias, url][]`
-    function rpcUrls() external view returns (string[2][] memory);
+    function rpcUrls() external view returns (string[2][] memory urls);
     // Returns all rpc urls and their aliases as structs.
-    function rpcUrlStructs() external view returns (Rpc[] memory);
+    function rpcUrlStructs() external view returns (Rpc[] memory urls);
     // If the condition is false, discard this run's fuzz inputs and generate new ones.
-    function assume(bool) external pure;
-
+    function assume(bool condition) external pure;
     // Pauses gas metering (i.e. gas usage is not counted). Noop if already paused.
     function pauseGasMetering() external;
     // Resumes gas metering (i.e. gas usage is counted again). Noop if already on.
     function resumeGasMetering() external;
 }
 
 interface Vm is VmSafe {
-    // Sets block.timestamp (newTimestamp)
-    function warp(uint256) external;
-    // Sets block.height (newHeight)
-    function roll(uint256) external;
-    // Sets block.basefee (newBasefee)
-    function fee(uint256) external;
-    // Sets block.difficulty (newDifficulty)
-    function difficulty(uint256) external;
+    // Sets block.timestamp
+    function warp(uint256 newTimestamp) external;
+    // Sets block.height
+    function roll(uint256 newHeight) external;
+    // Sets block.basefee
+    function fee(uint256 newBasefee) external;
+    // Sets block.difficulty
+    function difficulty(uint256 newDifficulty) external;
     // Sets block.chainid
-    function chainId(uint256) external;
-    // Stores a value to an address' storage slot, (who, slot, value)
-    function store(address, bytes32, bytes32) external;
+    function chainId(uint256 newChainId) external;
+    // Stores a value to an address' storage slot.
+    function store(address who, bytes32 slot, bytes32 value) external;
```

Comment:
```suggestion
    function store(address target, bytes32 slot, bytes32 value) external;
```

Reply:
`target` is definitely a good fit in this context.